packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -48,6 +51,41 @@ class MemoryController extends DisposableController
 
   static const String logFilenamePrefix = 'memory_log_';
 
+  bool showHeatMap = true;
+
+  final List<Snapshot> snapshots = [];
+
+  Snapshot get lastSnapshot => snapshots.isNotEmpty ? snapshots.last : null;
+
+  /// Notifies that the source of the memory feed has changed.
+  ValueListenable get selectedSnapshotNotifier => _selectedSnapshotNotifier;
+
+  final _selectedSnapshotNotifier = ValueNotifier<String>('');
+
+  set selectedSnapshotTimestamp(String snapshotTimestamp) {
+    _selectedSnapshotNotifier.value = snapshotTimestamp;
+  }
+
+  String get selectedSnapshotTimestamp => _selectedSnapshotNotifier.value;
+
+  HeapGraph theGraph;
```

Comment:
nit: rename from `theGraph` to `heapGraph`

Reply:
Done